ncm-opennebula/src/main/resources/user.tt

```diff
@@ -1,13 +1,14 @@
-[% FOR pair IN user.pairs -%]
-[%-    SWITCH pair.key -%]
-[%         CASE 'ssh_public_key' -%]
-[%             pair.key FILTER upper %] = "
-[%-             FOREACH item IN user.${pair.key} -%]
-[%                  item %]
-[%              END -%]"
-[%         CASE 'user' -%]
-[%             IF pair.value != 'oneadmin' -%]
+[% FOR pool IN user.pairs -%]
+[%     FOR pair IN pool.value.pairs %]
+[%-        SWITCH pair.key -%]
+[%             CASE 'ssh_public_key' -%]
+[%                 pair.key FILTER upper %] = "
+[%-                 FOREACH item IN pair.value -%]
+[%                      item %]
+[%                  END -%]"
```

Comment:
That is done on purpose, we don't want to include some values in the template as example the user's password.